audbackend/core/artifactory.py

```diff
@@ -154,6 +154,38 @@ def __init__(
         )
         self._repo = path.find_repository_local(self.repository)
 
+        self.extensions = []
+        r"""Custom extensions.
+
+        On Artifactory files are stored
+        under the following structure:
+        ``'.../<name-wo-ext>/<version>/<name-wo-ext>-<version>.<ext>'``.
+        By default,
+        the extension
+        ``<ext>``
+        is set the string after the last dot.
+        I.e.,
+        the backend path
+        ``'.../file.tar.gz'``
+        will translate into
+        ``'.../file.tar/1.0.0/file.tar-1.0.0.gz'``.
+        However,
+        by defining custom extensions
+        it is possible to overwrite
+        the default behavior.
+        E.g.,
+        with
+        ``backend.extensions.append('tar.gz')``
+        it is ensured that
+        ``'tar.gz'``
+        will be recognized as an extension
+        and the backend path
+        ``'.../file.tar.gz'``
+        will then translate into
+        ``'.../file/1.0.0/file-1.0.0.tar.gz'``.
+
+        """
```

Comment:
From the docstring one cannot easily infer the default value of `extensions` and that the user is supposed to provide a list of additional extensions. I would simply add 1-2 sentences stating this.

Reply:
The default value is an empty list and in the docstring we show an example how to add an extension to it, so I guess it should be fine.

Reply:
No longer relevant since we removed `Artifactory.extensions`.